.github/workflows/docker.yml

```diff
@@ -132,3 +134,16 @@ jobs:
             BUILD_IMAGE=${{ github.event_name == 'release' && github.ref_name || steps.metadata.outputs.tags }}
             BUILD_SOURCE_REF=${{ github.ref_name }}
             BUILD_SOURCE_COMMIT=${{ github.sha }}
+
+      - name: Extract geodata
+        if: ${{ github.event_name == 'release' && matrix.image == 'immich-server' }}
+        run: |
+          id="$(docker create ghcr.io/${{ github.repository_owner }}/${{ matrix.image }}:${{ github.ref_name }})"
+          docker cp "$id":/usr/bin/resources geodata
+          zip -r geodata.zip geodata
+
+      - name: Upload geodata as release asset
+        if: ${{ github.event_name == 'release' && matrix.image == 'immich-server' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          files: geodata.zip
```

Comment:
I assume calling it this way means that the zip file gets added to the existing release? Is that behaviour documented anywhere?

Reply:
I think I found a hint to that on Stackoverflow? The action README is a bit vague.
I verified this behavior in a fork though: https://github.com/jvanbruegge/immich/releases/tag/v1.108.0
The zip was attached by the action

Reply:
I've [opened a PR](https://github.com/softprops/action-gh-release/pull/474) to document this behavior